.github/workflows/quic.yml

```diff
@@ -0,0 +1,56 @@
+name: QUIC
+
+on:
+  push:
+    branches: ['main']
+    paths: ['.github/workflows/quic.yml', 'curl/**', 'nginx/**']
+  pull_request:
+    paths: ['.github/workflows/quic.yml', 'curl/**', 'nginx/**']
+  schedule:
```

Comment:
Don't understand this: Why run this on schedule, too? Isn't it sufficient when pushed to "main"? What changes in the code base happen afterwards warranting a re-run?

Reply:
It uses oqs-provider to generate certificates for nginx. If the scheduled task fails, it indicates a compatibility issue between oqs-boringssl and oqs-provider.

Reply:
Ah, very correct -- good idea! And these potential failures would then be(come) visible at the new QUIC status badge in the README?

Reply:
Yes!